experiments/AtmosLES/stable_bl_les.jl

```diff
@@ -81,4 +90,19 @@ function main()
     )
 end
 
+param_args = ArgParseSettings(autofix_names = true)
+add_arg_group!(param_args, "ParameterVersion")
+@add_arg_table! param_args begin
+    "--CP-Version"
+    help = "specify CLIMAParameters version"
+    arg_type = Union{String, Nothing}
+    default = nothing
+end
+
+parsed_args = parse_args(ARGS, param_args)
+if !isnothing(parsed_args["CP_Version"])
+    filename = "clima_param_defs_$(parsed_args["CP_Version"]).jl"
```

Comment:
So here: filename would be the path to the file.  include doesn't have to be a relative path (it can be an absolute path))